### PR TITLE
Mini Upstream Merge: woops edition

### PR DESCRIPTION
**Pull request by ghost**
this should fix all the botched mirrors and git fuckery

closes #3035
closes #3034
closes #3031
closes #3029

Should also (fucking finally) fix the changelogs

### CODE REVIEW

**Comment by ghost** [comment]
i accidently pushed my merge fixing to my fork's master branch

File: code/game/gamemodes/gang/gang.dm [hunk 1]
```diff
@@ -1,359 +0,0 @@
-//gang.dm
```
Comment (ghost) [inline code/game/gamemodes/gang/gang.dm:1]: ?????????????????????????????????????????????????????

**Review by ghost — DISMISSED** [review]

File: code/game/gamemodes/gang/gang.dm (hunk 1 above)
Comment (ghost) [reply code/game/gamemodes/gang/gang.dm:1]: did you forget https://github.com/HippieStation/HippieStation/pull/3023 got merged or something

**Comment by GuyonBroadway** [comment]
fix the merge you made and I'll gofer it.

**Comment by ghost** [comment]
@GuyonBroadway that doesn't affect the pr in the slightest. after this is merged i can fix it with a forcepush to my fork

**Review by ghost — APPROVED** [review]

**Pull request merged**